src/index.js

```diff
@@ -165,13 +165,36 @@ const isHidden = function (node, displayCheck) {
   return false;
 };
 
+// form fields (nested) inside a disabled fieldset are not focusable/tabbable
+const isInsideDisabledFieldset = function (node) {
+  if (
+    isInput(node) ||
+    node.tagName === 'SELECT' ||
+    node.tagName === 'TEXTAREA' ||
+    node.tagName === 'BUTTON'
+  ) {
+    let parentNode = node.parentElement;
+    while (parentNode) {
```

Comment:
My first thought was to use a query, but I don't think there's a selector that can do this, especially because the disabled fieldset could be one that is an ancestor far removed from where the node actually is in the hierarchy within it.

Reply:
Fair enough. :where selector would have been nice

Reply:
Let's say we used this as the selector, `fieldset:not([disabled]) :where(input, button, textarea, select)` It still selects an input inside an enabled fieldset, which is nested inside a disabled fieldset -- and the fact the parent fieldset is disabled means that input will be disabled, even though it's nearest parent is an enabled fieldset.

Reply:
without taking the `<legend>` rule this would have targeted all buttons that have no `[disabled]` attribute and aren't nested under a `[disabled]` `<fieldset>`:

```css
button:not(
  [disabled],
  :where(fieldset[disabled] *)
)
```

adding buttons that are nested under the first `<legend>` but not under a any other more nested `<legend>` is more difficult.

Reply:
yes, this is difficult :(